pptx_blueprint/__init__.py

```diff
@@ -42,14 +42,57 @@ def replace_picture(self, label: str, filename: _Pathlike) -> None:
         """
         pass
 
-    def replace_table(self, label: str, data) -> None:
+    def replace_table(self, label: str, data, header=False, rownames=False):
```

Comment:
We allow use kwargs only for `header` and `rownames`:
```python
def replace_table(self, label: str, data, *, header=False, rownames=False):
```
In my opinion it's better to use `None` instead of `False`. When using `False` as default value I assume a Boolean value is required. `None` is no valid data fore these arguments and therefore it should be fine to use it. Using an `Undefined()` object could be an option, too. But I think this is not necessary in this case.